tests/Main.hs

```diff
@@ -40,7 +40,10 @@ cabalClashBinDir :: IO String
 cabalClashBinDir = makeAbsolute rel_path
  where
   rel_path = printf templ platform ghcVersion3 (VERSION_clash_ghc :: String)
-  platform = "x86_64-linux" :: String -- XXX: Hardcoded
+  platform :: String -- XXX: Hardcoded
```

Comment:
Shouldn't the comment have been removed? Or do you still consider this hardcoding, even though softer? :-)

I could add it to a new incarnation of `random-stuff`.

Reply:
I still consider it hardcoding, preferably we would get this string through some API call in the `Cabal` library.